src/Templates/src/templates/maui-mobile/MauiApp.1.csproj

```diff
@@ -48,7 +56,23 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="MS_EXT_VERSION" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="$(MicrosoftExtensionsPackageVersion)" />
```

Comment:
Hmm is this change deliberate? These templates have some funky pre-processing that is applied as part of the build to substitute a specific version that ends up baked into the shipped template.

Reply:
Oof, accidentally got modified, thanks!